Post/smaller the parameter smaller the bugs (#11)
rakutentech/engineering-blog: content/authors/bater/_index.md

---
title: Bater Chen
bio: |

>   Love to write the short and simple code to implement the meaningful project in Ruby.

Reviewer: ```suggestion
  Love to write short and simple code for meaningful projects in Ruby.
```
